src/holoscan_cli/common/argparse_types.py

```diff
@@ -84,6 +84,10 @@ def valid_existing_path(path: str) -> Path:
     path = os.path.expanduser(path)
     file_path = Path(path).absolute()
     if file_path.exists():
+        if os.path.isdir(file_path):
+            if os.listdir(path):
```

Comment:
Any reason why `path` variable instead of the `file_path` is used here? The latter is a Path obj and has the is_dir(), and iterdir() functions.

Reply:
Thanks! I fixed it.